controllers/v2/unavailability/api.py

```diff
@@ -59,8 +61,8 @@ def get(self, user_id):
             else:
                 return jsonify({'userID': user_id, 'success': False}), 400
 
-    @requires_auth
```

Comment:
Do not remove these. Make a global variable that allows you to disable these checks.